ci/memo.sh

```diff
@@ -4,5 +4,7 @@ cd "$(dirname "$0")/.."
 
 set -e
 
+./do.sh update
```

Comment:
We'll want to remove this line: https://github.com/solana-labs/solana-program-library/blob/f57149340401051b2e1c365410d48ea39b3eb09d/.travis.yml#L27

Reply:
Why?

Reply:
Well, alternately, remove `./do.sh update` from the beginning of each ci script.
As it is, `./do.sh update` will run twice in a row for each job.